src/node_options.cc

```diff
@@ -1528,6 +1545,81 @@ void GetEmbedderOptions(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(ret);
 }
 
+// This function returns a map containing all the options available
+// as NODE_OPTIONS and their input type
+// Example --experimental-transform types: kBoolean
+// This is used to determine the type of the input for each option
+// to generate the config file json schema
+void GetEnvOptionsInputType(const FunctionCallbackInfo<Value>& args) {
+  Isolate* isolate = args.GetIsolate();
+  Local<Context> context = isolate->GetCurrentContext();
+  Environment* env = Environment::GetCurrent(context);
+
+  if (!env->has_run_bootstrapping_code()) {
+    // No code because this is an assertion.
+    return env->ThrowError(
+        "Should not query options before bootstrapping is done");
```

Comment:
This needs to throw an error with an appropriate `code`

Reply:
created and documented `[THROW_ERR_OPTIONS_BEFORE_BOOTSTRAPPING](https://github.com/nodejs/node/pull/57016/commits/2cff1991c54660aaa01665566dc83a09fa890ac8)`